tests/model_registry/utils.py

```
@@ -264,3 +264,55 @@ def generate_random_name(prefix: str, length: int = 8) -> str:
 
 def generate_namespace_name(file_path: str) -> str:
     return (file_path.removesuffix(".py").replace("/", "-").replace("_", "-"))[-63:].split("-", 1)[-1]
+
+
+def create_model_registry_instance(
```

Pylint (3.3.7)

[refactor] Too many arguments (13/5)

(R0913)